config/etc/system-image/writable-paths

```diff
@@ -72,7 +72,7 @@
 /etc/network/if-up.d                    auto                    persistent  transition  none
 /etc/NetworkManager/system-connections  auto                    persistent  none        none
 # snappy security policy, etc
-/var/lib/snapd                         auto                    persistent  transition  none
+/var/lib/snapd                         auto                     synced      none        none
```

Comment:
There's an extra space here before `synced`

Reply:
nope, the former "persistent" was wrongly indented (the file uses spaces for indentation) ...